packages/terra-data-grid/src/FlowsheetDataGrid.jsx

```diff
@@ -109,10 +109,6 @@ const propTypes = {
    * Boolean to show/hide column headers. By default, it is set to `true` and column headers are visible.
    */
   hasVisibleColumnHeaders: PropTypes.bool,
-  /**
-   * Bounding container for the flowsheet grid, will use window if no value provided.
-   */
-  boundingRef: PropTypes.func,
```

Comment:
Is there a reason this prop is being removed? Because this would be a breaking change.

Reply:
The reason this is being removed is that we should not be forcing consumers to use a bounding ref in order to make the section headers sticky. Instead we are using the internal table container to control the sticky behavior. After this PR the sticky header will continue as before, but the component will not require this div anymore.

Reply:
Makes sense +1